index.html

```diff
@@ -7,6 +7,6 @@
 <body>
     <div class="main">
         <h1>ObieSource</h1>
-        <a href="/membersdir.html">Browse our members directory!</a>
+        <a title="Directory of club members" href="/membersdir.html">Browse our members directory!</a>
```

Comment:
I would also suggest removing the `title` attributes that you added. (here, in `member.html`, and in `membersdir.html`)